[AGENTLESS][SEC-17683] Add Azure onboarding to Agentless documentation (#26786)
DataDog/documentation: content/en/security/cloud_security_management/setup/agentless_scanning/_index.md

<div class="alert alert-warning">Agentless Scanning for Cloud Security Management is not supported for your selected <a href="/getting_started/site">Datadog site</a> ({{< region-param key="dd_site_name" >}}).</div>
{{< /site-region >}}

> Agentless Scanning provides visibility into vulnerabilities that exist within your cloud infrastructure,
> without requiring you to install the Datadog Agent.
> 
> See the [CSM Agentless Scanning][12] page to learn more about its capabilities and how it works.

Reviewer: ```suggestion
Agentless Scanning provides visibility into vulnerabilities that exist within your cloud infrastructure, without requiring you to install the Datadog Agent. To learn more about Agentless Scanning's capabilities and how it works, see the [Agentless Scanning][12] docs.
```

## Prerequisites

Before setting up Agentless Scanning, ensure the following prerequisites are met:

> - **Remote Configuration**: [Remote Configuration][3] is required to enable Datadog to send information to Agentless scanners, such as the cloud resources to scan.

Reviewer: ```suggestion
- **Remote Configuration**: [Remote Configuration][3] is required to enable Datadog to send information to Agentless scanners, such as which cloud resources to scan.
```

Use the AWS CloudFormation template to create a CloudFormation stack. The template includes the IAM permissions required to deploy and manage Agentless scanners. For more information, see [Setting up Agentless Scanning using AWS CloudFormation][11].

### Azure Resource Manager

> Use the Azure Resource Manager template to deploy the Agentless Scanner. The template includes the rule definitions required to deploy and manage Agentless scanners. For more information, see [Setting up Agentless Scanning using Azure Resource Manager][13].

Reviewer 2: ```suggestion
Use the Azure Resource Manager template to deploy the Agentless Scanner. The template includes the role definitions required to deploy and manage Agentless scanners. For more information, see [Setting up Agentless Scanning using Azure Resource Manager][13].
```
